### DISCUSSION TITLE
"Print" via Meshtastic encrypted mesh network.

### DISCUSSION THREAD

**Discussion by btcpuertorico** [selfcustody/krux#313] [Ideas]
Hello,

Just thinking that it may be possible to transmit  ̶m̶n̶e̶m̶o̶n̶i̶c̶, xpub,  ̶w̶a̶l̶l̶e̶t̶ ̶b̶a̶c̶k̶u̶p̶, and signed PSBT also via encrypted mesh network Meshtastic. Please read (and confirm it can be done ;) ) : https://meshtastic.org/docs/configuration/module/serial/

Many of the devices have or can be configured with an e-ink display. Also a module for the Meshtastic apps can be developed to display received communication as QR code at the receiving end.

This can greatly expand the freedom tech toolset to work around heavily adversarial scenarios.

**Comment by tadeubas** [comment]
Never display your mnemonic or transmit it in an uncontrolled medium. Now for the transmission of other non-sensitive information, I think it is a possibility to connect and configure a LoRa card like this on the photo on the Amigo's device. But perhaps this could be something to be added not to the Krux firmware itself, but more as an additional application, to be run only when needed

![image](https://github.com/selfcustody/krux/assets/5518496/def95b0f-0065-4e27-a499-86ecdabe890b)

**Comment by btcpuertorico** [comment]
Oh! that was copy/paste abuse on my part. Not supposed to include mnemonic. Yes, that may be the way. Similar to the [applet](https://finance.yahoo.com/news/gotenna-samourai-wallet-mobile-app-163800571.html) Samourai did with Gotenna but using fully open-source software.

> Never display your mnemonic or transmit it in an uncontrolled medium. Now for the transmission of other non-sensitive information, I think it is a possibility to connect and configure a LoRa card like this on the photo on the Amigo's device. But perhaps this could be something to be added not to the Krux firmware itself, but more as an additional application, to be run only when needed
> 
> ![image](https://private-user-images.githubusercontent.com/5518496/301826277-def95b0f-0065-4e27-a499-86ecdabe890b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcwNDI2NjMsIm5iZiI6MTcwNzA0MjM2MywicGF0aCI6Ii81NTE4NDk2LzMwMTgyNjI3Ny1kZWY5NWIwZi0wMDY1LTRlMjctYTQ5OS04NmVjZGFiZTg5MGIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMDRUMTAyNjAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Mzg5MWEyZWI1OGI5OGY5NjAzODU0NzU5N2Y5OWY3ZjFjN2UyMjc4MjI4ZmIwM2JkZDU3NjliZTYzZmJiYTVhNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.M0w_vrchwpO7LvsMKf_FvVkDFZEOLSAuX6nfhIdPUbw)
